CMakeLists.txt

```diff
@@ -1,5 +1,9 @@
 cmake_minimum_required(VERSION 3.22)
-project(faker-cxx LANGUAGES CXX)
+project(faker-cxx
+        LANGUAGES CXX
+        VERSION 2.0.0
```

Comment:
This VERSION is consumed by CMake to generate everything related to the project version, Version.h, ConfigVersion.cmake and so version for shared libraries (faker-cxx.so.2.0.0)